integTests/main_test.go

```diff
@@ -15,11 +16,9 @@ import (
 	"github.com/xdblab/xdb-golang-sdk/integTests/worker"
 )
 
-var testDatabaseName = fmt.Sprintf("tst%v", time.Now().UnixNano())
```

Comment:
Remove using random new database for integ test because our source connector has to specify a test database name. This need more work to make sure it works together.